### ISSUE TITLE
 Callback was already called

### ISSUE THREAD

**Issue by MeijeSibbel** [Storj/core-cli#102]
### Package Versions

Replace the values below using the output from `npm list storj`. Use ```npm list -g storj``` if installed globally.

```
storj-bridge@5.5.2 /home/gordon/Code/storj-bridge
└── Storjcli: 4.0.0 | Core: 6.3.2
```

Replace the values below using the output from `node --version`.

```
v6.10.0
```

### Expected Behavior

Please describe the program's expected behavior. Include an example of your
usage code in the back ticks below if applicable.

```
Upload is successful   
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
[Thu Mar 16 2017 22:16:28 GMT+0100 (W. Europe Standard Time)] [info]   Hash for this shard is: d15bc15e42a21265b14ccc9434ca1a11ad1c677c
[Thu Mar 16 2017 22:16:32 GMT+0100 (W. Europe Standard Time)] [info]   Audit generation for shard done.
[Thu Mar 16 2017 22:16:32 GMT+0100 (W. Europe Standard Time)] [info]   Waiting on a storage offer from the network...
[Thu Mar 16 2017 22:16:32 GMT+0100 (W. Europe Standard Time)] [info]   Querying bridge for contract for d15bc15e42a21265b14ccc9434ca1a11ad1c677c (retry: 0)
[Thu Mar 16 2017 22:16:33 GMT+0100 (W. Europe Standard Time)] [info]   Contract negotiated with: {"userAgent":"6.3.2","protocol":"1.1.0","address":"64.137.206.228","port":
,"nodeID":"d1672d5f23e53304e1cc0a2819c91274fbd05ba5","lastSeen":1489656508530}
[Thu Mar 16 2017 22:16:34 GMT+0100 (W. Europe Standard Time)] [info]   Shard transfer completed! 10 remaining...
:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\node_modules\async\dist\async.js:840
       if (fn === null) throw new Error("Callback was already called.");
                        ^

rror: Callback was already called.
   at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\node_modules\async\dist\async.js:840:32
   at BridgeClient._shardTransferComplete (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:778:12)
   at EventEmitter.<anonymous> (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:753:10)
   at EventEmitter.g (events.js:291:16)
   at emitNone (events.js:86:13)
   at EventEmitter.emit (events.js:185:7)
   at Transform.<anonymous> (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:869:9)
   at emitNone (events.js:91:20)
   at Transform.emit (events.js:185:7)
   at finishMaybe (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\readable-stream\lib\_stream_writable.js:475:14)
   at endWritable (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\readable-stream\lib\_stream_writable.js:485:3)
   at Transform.Writable.end (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\readable-stream\lib\_stream_writable.js:455:41)
   at ReadStream.onend (_stream_readable.js:511:10)
   at ReadStream.g (events.js:291:16)
   at emitNone (events.js:91:20)
   at ReadStream.emit (events.js:185:7)

:\Users\Dani Marion>
```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. upload a file sequence with /*
2. ...
3. ...

**Comment by braydonf** [comment]
This is duplicate of https://github.com/Storj/core/issues/670
